updater/lib/dependabot/service.rb

```diff
@@ -49,8 +49,7 @@ def record_update_job_error(error_type:, error_details:, dependency: nil)
       client.record_update_job_error(error_type: error_type, error_details: error_details)
     end
 
-    def record_update_job_unknown_error(error_type:, error_details:, dependency: nil)
-      @errors << [error_type.to_s, dependency]
+    def record_update_job_unknown_error(error_type:, error_details:)
```

Comment:
I removed adding to `@errors` here because otherwise the PR summary displayed in job logs was showing two duplicated errors. And that was making one spec failing that checked for a "Dependabot encountered '1' error" log.

Reply:
thank you so much. this was getting super hard to crack it for me atleast.